### ISSUE TITLE
Unable to create users immediatly after the the app is created using the WebAPI

### ISSUE THREAD

**Issue by MikeAlhayek** [sendbird/sendbird-platform-sdk-csharp#2]
I am trying to integrate my app with Sendbird using REST WebAPI.

When I create the new application, I get the `AppId` in respond along with the `ApiToken` which is great. The returned `AppId` is usually is in all caps `NEW-APPLICATION-ID`. If I send a second request to create new users immediately after the app is created, I get the following error

> Name or service not known (api-new-application-id.sendbird.com:443)

The problem here is that when using `C#` to create a `Uri()` to use for the `HttpClient`, the URI is converted to all lowercase. Because domains are case-insensitive by nature.

However, if I wait a white and then try to create users using the same host `api-new-application-id.sendbird.com:443` it works with no probelm. It's like the server requires time delay to be aware of the case insensitive domain.

Is there a work around for this problem?

**Comment by supertopoz** [comment]
@MikeAlhayek Hi there. 

My recommendation would be to wait sometime for the application set up to be completed before running any requests to it. In normal cases, it should take several seconds or more for the new application set up to complete. What do you think?

**Comment by MikeAlhayek** [comment]
@supertopoz Yes that is the approach I already took as a workaround. But, do you have any idea how long should we wait? I have to sync up the users when after creating the application.

Anyway, I don't think the applications API should respond with success until the process is complete. If the API on your end fires some sort of background job which does internal work. Then maybe the API should have a call back url to call when the application is ready for usage. In other words, add a new parameter (ex, `callback_url`) where your API will post-back to after the application is completely setup and ready for action. This way we can do follow up tasks like synchronize users, create default channels or anything else that maybe needed.

**Comment by supertopoz** [comment]
@MikeAlhayek Hi again. 

Can I ask are you using this C# SDK to create the apps or REST webapi as you mention in the first case, please? 

There is a way to call to the newly created app without needing the app_id, but I couldn't see the code in the this SDK so wondering if you are using this end point directly. 
https://sendbird.com/docs/chat/platform-api/v3/organization/managing-application/create-an-application#1-create-an-application

**Comment by MikeAlhayek** [comment]
@supertopoz i am using the REST API directly without any SDK.

To create the application I use this endpoint

https://sendbird.com/docs/chat/platform-api/v3/organization/managing-application/create-an-application#1-create-an-application

Which works and return the appId and the auth token.

Then the second request would be to use the newly created AppId to create users using the following endpoint

https://sendbird.com/docs/chat/platform-api/v3/user/creating-users/create-a-user

**Comment by supertopoz** [comment]
@MikeAlhayek I sent you an email with a possible solution.
